### PR TITLE
3912 wagtail 28

### PR DESCRIPTION
**Pull request by easherma**
Upgrades Wagtail, does some minor tweaks to maintain compatibility with embeds, and also hides the 'Reports' section of the menu, which will show you any pages you've locked from editing. 

Deployed at: https://dashboard.heroku.com/apps/joplin-pr-3912-wagtail-28

### CODE REVIEW

**Comment by robbobfrh84** [comment]
Hey @easherma, I'm curious, what is `reports` in the cinfig is related to? Also, is `WAGTAILEMBEDS_RESPONSIVE_HTML = True` new to this release or are we adding it just now?

File: joplin/base/wagtail_hooks.py [hunk 1]
```diff
@@ -61,7 +61,8 @@ def configure_main_menu(request, menu_items):
                      [
                          'explorer',
                          'settings',
-                         'snippets'
+                         'snippets',
+                         'reports'
```
Comment (easherma) [inline joplin/base/wagtail_hooks.py]: This is a new thing that wagtail has, which we don't do anything with, so I'm assuming we'll want to keep hiding it.

File: joplin/settings.py [hunk 2]
```diff
@@ -215,6 +215,7 @@
 DEBUG_TOOLBAR = bool(strtobool(os.environ.get('DEBUG_TOOLBAR', str(False))))
 MONITOR_PERFORMANCE = bool(strtobool(os.environ.get('MONITOR_PERFORMANCE', str(False))))
 
+WAGTAILEMBEDS_RESPONSIVE_HTML = True
```
Comment (easherma) [inline joplin/settings.py]: maintains previous compatibility, we still use these classes in janis (or at least they _are_ in janis)

**Comment by easherma** [comment]
> Hey @easherma, I'm curious, what is `reports` in the cinfig is related to? Also, is `WAGTAILEMBEDS_RESPONSIVE_HTML = True` new to this release or are we adding it just now?

@robbobfrh84 see:
https://docs.wagtail.io/en/latest/releases/2.8.html#responsive-html-for-embeds-no-longer-added-by-default

File: joplin/base/wagtail_hooks.py (hunk 1 above)
Comment (robbobfrh84) [reply joplin/base/wagtail_hooks.py]: Curious what the reports.. report. could be really interesting.

File: joplin/settings.py (hunk 2 above)
Comment (robbobfrh84) [reply joplin/settings.py]: Gotchya!👍
... also, you responded to both of these questions in like 10 seconds.

File: joplin/base/wagtail_hooks.py (hunk 1 above)
Comment (easherma) [reply joplin/base/wagtail_hooks.py]: Check it out in https://odd-tail-bakery.herokuapp.com/admin/

admin : admin
PW: changeme

**Pull request merged**